source/common/quic/udp_gso_batch_writer.h

```diff
@@ -92,13 +94,20 @@ class UdpGsoBatchWriter : public quic::QuicGsoBatchWriter, public Network::UdpPa
 
 class UdpGsoBatchWriterFactory : public Network::UdpPacketWriterFactory {
 public:
+  std::string name() const override { return "envoy.udp.writer.factory.gso"; }
   Network::UdpPacketWriterPtr createUdpPacketWriter(Network::IoHandle& io_handle,
                                                     Stats::Scope& scope) override;
+  ProtobufTypes::MessagePtr createEmptyConfigProto() override {
+    return std::make_unique<
+        envoy::extensions::quic::udp_packet_writer::v3::UdpGsoBatchWriterFactory>();
```

Comment:
Wait a minute, where is this proto defined?

Reply:
Bah! Forgot to do a git add. Done.